src/test/java/com/google/api/codegen/testdata/discoveries/java/java_appengine.v1beta5.json.baseline

```diff
@@ -8,7 +8,8 @@
  *    https://github.com/google/google-api-java-client.
  *    On other build systems, you can add the jar files to your project from
  *    https://developers.google.com/resources/api-libraries/download/appengine/v1beta5/java
- * 3. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from
+ * 3. This sample uses Application Default Credentials for Auth.
```

Comment:
Oh, I don't mean to break the URL, of course. But the text preceding the URL in 1 appears long.